specification/servicebus/resource-manager/readme.go.md

```diff
@@ -23,12 +23,22 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2022-10-preview
   - tag: package-2021-06-preview
   - tag: package-2018-01-preview
   - tag: package-2017-04
   - tag: package-2015-08
 ```
 
+### Tag: package-2022-10-preview and go
+
+These settings apply only when `--tag=package-2022-10-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-10-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2022-10-01-preview/$(namespace)
+```
+
```

Comment:
These sections are for the deprecated track 1 SDKs, therefore we no longer need to update it. Could you please revert these changes